app/styles/currency-input.less

```diff
@@ -47,6 +47,37 @@
       outline: none;
     }
 
+    &:focus-within {
+      border: 1px solid var(--color-border-default);
+    }
+  }
+  .currency-input-onlycurrency {
+    padding-right: 0px;
+    .currency-selector {
+      height: 100%;
+      display: flex;
+      align-items: center;
+      padding-right: var(--spacing-px-12);
+      margin-right: 0px;
+      width: 100%;
+    }
+
+    .phone-prefix {
+      white-space: nowrap;
+      .text-color-default-light;
+      .margin-right-xxx-sm;
+    }
```

Comment:
`.phone-prefix` is not used in currency-input.
Be careful when copy/pasting code :)